tests/test_self.py

```diff
@@ -1154,6 +1154,14 @@ def test_fail_on_exit_code(self, args, expected):
         # and errors that are generated they don't affect the exit code.
         self._runtest([path, "--fail-under=-10"] + args, code=expected)
 
+    def test_one_module_fatal_error(self):
+        """
+        Fatal errors in one of several modules linted still exits non-zero.
+        """
+        valid_path = join(HERE, "conftest.py")
+        invalid_path = join(HERE, "garbagePath.py")
+        self._runtest([valid_path, invalid_path], code=1)
```

Comment:
Isn't expected code `0` here? I haven't looked closely at `_runtest` yet so I might be missing something.

Reply:
0 would indicate success. A fatal error is raised linting `garbagePath.py`, so now (with this PR) we exit with 1.

Reply:
My bad.. Sorry! Mixed up 0 and 1. Thanks!